rust/src/common/config.rs

```diff
@@ -27,7 +27,12 @@ const DFS_CLIENT_FAILOVER_RANDOM_ORDER: &str = "dfs.client.failover.random.order
 // Viewfs settings
 const VIEWFS_MOUNTTABLE_PREFIX: &str = "fs.viewfs.mounttable";
 
-#[derive(Debug)]
+const DFS_CLIENT_WRITE_REPLACE_DATANODE_ON_FAILURE_ENABLE_KEY: &str =
+    "dfs.client.block.write.replace-datanode-on-failure.enable";
+const DFS_CLIENT_WRITE_REPLACE_DATANODE_ON_FAILURE_POLICY_KEY: &str =
+    "dfs.client.block.write.replace-datanode-on-failure.policy";
```

Comment:
Should we go ahead and add `DFS_CLIENT_WRITE_REPLACE_DATANODE_ON_FAILURE_BEST_EFFORT_KEY` as well? Seems like you already implemented the behavior for it

Reply:
make sense.